simtools/testing/assertions.py

```diff
@@ -0,0 +1,43 @@
+"""Functions asserting certain conditions are met (used e.g., in integration tests)."""
+
+import json
+import logging
+from pathlib import Path
+
+import yaml
+
+_logger = logging.getLogger(__name__)
+
+
+def assert_file_type(file_type, file_name):
+    """
+    Assert that the file is of the given type.
+
+    Parameters
+    ----------
+    file_type: str
+        File type (json, yaml).
+    file_name: str
+        File name.
+
+    """
+    if file_type == "json":
+        try:
+            with open(file_name, encoding="utf-8") as file:
+                json.load(file)
+            return True
+        except (json.JSONDecodeError, FileNotFoundError):
+            return False
+    if file_type in ("yaml", "yml"):
+        if Path(file_name).suffix[1:] not in ("yaml", "yml"):
```

Comment:
I thought that if the file has no suffix, `file.suffix[1:]` would raise an error of the index being out of bound, but apparently that doesn't happen with strings and this is safe.